### PR TITLE
testing/php7-phalcon: new aport

### PR DESCRIPTION
**Pull request by vakartel**
High performance, full-stack PHP framework delivered as a C extension
https://github.com/phalcon/cphalcon
https://phalconphp.com

### CODE REVIEW

File: testing/php7-phalcon/APKBUILD [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+_pkgext=phalcon
+_pkgreal=cphalcon
+pkgver=3.3.2
+_pkgver=${pkgver/_rc/RC}
+pkgrel=0
+pkgdesc="High performance, full-stack PHP framework delivered as a C extension"
+url="https://github.com/phalcon/cphalcon"
+arch="all"
+options="!check" # there is no module tests
+license="BSD"
```
Comment (andypost) [inline testing/php7-phalcon/APKBUILD]: It should be `BSD-3-Clause` according to https://spdx.org/licenses/BSD-3-Clause.html and looking at https://github.com/phalcon/cphalcon/blob/master/LICENSE.txt

File: testing/php7-phalcon/APKBUILD [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+pkgname=php7-phalcon
+_pkgext=phalcon
+_pkgreal=cphalcon
+pkgver=3.3.2
+_pkgver=${pkgver/_rc/RC}
+pkgrel=0
+pkgdesc="High performance, full-stack PHP framework delivered as a C extension"
+url="https://github.com/phalcon/cphalcon"
+arch="all"
+options="!check" # there is no module tests
```
Comment (andypost) [inline testing/php7-phalcon/APKBUILD]: not exactly true https://travis-ci.org/phalcon/cphalcon

Comment (andypost) [reply testing/php7-phalcon/APKBUILD]: better to said `require extra services to run tests` https://github.com/phalcon/cphalcon/blob/master/.travis.yml#L37

**Comment by vakartel** [comment]
Ok, done it

**Comment by andypost** [comment]
squashed http://dup.pw/aports/fc673f31

**Pull request closed without merging**